src/DetailsView/components/failure-instance-panel-details.tsx

```diff
@@ -13,26 +14,47 @@ export type FailureInstancePanelDetailsProps = {
 };
 
 export const FailureInstancePanelDetails = NamedSFC<FailureInstancePanelDetailsProps>('FailureInstancePanelDetails', props => {
+    const getSnippetInfo = (): JSX.Element => {
+        if (!props.snippet) {
+            return (
+                <div className="failure-instance-snippet-empty-body"> Code snippet will auto-populate based on the CSS selector input.</div>
+            );
+        } else if (props.snippet.startsWith('No code snippet is mapped to:')) {
```

Comment:
We should explore defining the "did I get an error" state separately from the UX string we'd like to use. That way the `path-snippet-controller` and this line both don't know about the UX string so changing it is easier in the future.